### PR TITLE
Sync Serverless CI

### PR DESCRIPTION
**Pull request by serverless-qe**
Sync Serverless CI using openshift-knative/hack.

### CODE REVIEW

File: ci-operator/jobs/openshift-knative/serving/openshift-knative-serving-release-v1.11-presubmits.yaml [hunk 1]
```diff
@@ -58,6 +58,77 @@ presubmits:
         secret:
           secretName: result-aggregator
     trigger: (?m)^/test( | .* )411-images,?($|\s.*)
+  - agent: kubernetes
+    always_run: false
```
Comment (skonto) [inline ci-operator/jobs/openshift-knative/serving/openshift-knative-serving-release-v1.11-presubmits.yaml:62]: Now this is correct.

File: ci-operator/jobs/openshift-knative/serving/openshift-knative-serving-release-v1.12-presubmits.yaml [hunk 2]
```diff
@@ -58,6 +58,77 @@ presubmits:
         secret:
           secretName: result-aggregator
     trigger: (?m)^/test( | .* )411-images,?($|\s.*)
+  - agent: kubernetes
+    always_run: false
```
Comment (skonto) [inline ci-operator/jobs/openshift-knative/serving/openshift-knative-serving-release-v1.12-presubmits.yaml:62]: Now this is correct.

**Review by ReToCode — APPROVED** [review]
/lgtm
/approve

@skonto  do you want to run the tests to see if everything works?

**Comment by skonto** [comment]
cc @mgencur now the PR is way larger :)

**Comment by mgencur** [comment]
> cc @mgencur now the PR is way larger :)

Nevermind :)

I'd say let's remove all configs older than 1.9. They're unsupported anyway.

**Comment by skonto** [comment]
```
error: resource mapping not found for name: "knative-eventing" namespace: "knative-eventing" from "STDIN": no matches for kind "KnativeEventing" in version "operator.knative.dev/v1alpha1"
ensure CRDs are installed first
```

**Comment by mgencur** [comment]
> Some branches and ocp versions eg. 4.10 do fail but they are not supported. @mgencur should we pj-rehearse ack?

No. Let's remove them in a follow-up PR.

**Comment by skonto** [comment]
> No. Let's remove them in a follow-up PR.

Ok are you going to do it? I created one PR to fix the hyperfoil thingy.

**Comment by mgencur** [comment]
> Ok are you going to do it? I created one PR to fix the hyperfoil thingy.

I can do it on Mon.

**Comment by skonto** [comment]
It is ok seems small anyway, https://github.com/openshift-knative/hack/pull/90, let me know if I missed anything.

**Comment by skonto** [comment]
@mgencur probably it is better to wait get all hack PRs merged ;)

**Comment by pierDipi** [comment]
/pj-rehearse pull-ci-openshift-knative-eventing-release-v1.11-411-test-encryp-9732689-aws-ocp-411

File: ci-operator/config/openshift-knative/eventing-kafka-broker/openshift-knative-eventing-kafka-broker-release-next__411.yaml [hunk 3]
```diff
@@ -421,10 +457,6 @@ tests:
         name: knative-eventing-kafka-broker-test-wathola-sender
       - env: KNATIVE_EVENTING_KAFKA_BROKER_SOURCE_IMAGE
         name: knative-eventing-kafka-broker-source-image
-      - env: KNATIVE_EVENTING_KAFKA_BROKER_DISPATCHER
-        name: knative-eventing-kafka-broker-dispatcher
-      - env: KNATIVE_EVENTING_KAFKA_BROKER_RECEIVER
-        name: knative-eventing-kafka-broker-receiver
```
Comment (pierDipi) [inline ci-operator/config/openshift-knative/eventing-kafka-broker/openshift-knative-eventing-kafka-broker-release-next__411.yaml:427]: /hold

Comment (pierDipi) [reply ci-operator/config/openshift-knative/eventing-kafka-broker/openshift-knative-eventing-kafka-broker-release-next__411.yaml:427]: I'm not sure why we've this deletion and what was changed in hack recently

Comment (pierDipi) [reply ci-operator/config/openshift-knative/eventing-kafka-broker/openshift-knative-eventing-kafka-broker-release-next__411.yaml:427]: I see here https://github.com/openshift-knative/hack/pull/84, we need to add a pattern

**Review by pierDipi — APPROVED** [review]
/lgtm
/approve
/unhold

**Pull request merged**